openlibrary/plugins/upstream/utils.py

```diff
@@ -1158,6 +1170,56 @@ def reformat_html(html_str: str, max_length: int | None = None) -> str:
         return ''.join(content).strip().replace('\n', '<br>')
 
 
+def get_isbn_10_and_13(isbns: str | list[str]) -> Isbns:
```

Comment:
A tuple is clearer than a data class in this case ; the function name already tells you what you're going to get. Adding a class for something kind of "meaningless" like a pair of isbn_10/isbn_13 adds extra abstraction/complexity.

```suggestion
def get_isbn_10_and_13(isbns: str | list[str]) -> tuple[list[str], list[str]]:
```

Reply:
Makes sense!